packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts

```diff
@@ -53,6 +53,13 @@ export class NoUsageData implements IUsageData, IFlowData {
     /* noop */
   }
 
+  /**
+   * Noop implementation of getSessionUuid
+   */
+  getSessionUuid(): string {
+    return '';
+  }
```

Comment:
todo?

Reply:
no, this is a mock UsageData, empty string is ok